### PR TITLE
Change fs to async readFile

### PR DESCRIPTION
**Pull request by humphd**
Hey,

I noticed that you were using a sync api, this fixes it to not block.

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -29,16 +30,21 @@ app.post('/api/phonenumbers/parse/file', upload.single('file'), (req, res) => {
...
-    var contents = fs.readFileSync(req.file.path);
-    var fileText = contents.toString('ascii');
-    var buf = Buffer.from(fileText, 'base64');
-    var numbers = buf.toString('ascii');
-    var numArr = numbers.split('\n');
-
-    var finalArr = numParser(numArr, res);
-
-    res.status(200).send(finalArr);
+//    var contents = fs.readFileSync(req.file.path);
```
Comment (apetov) [inline server.js]: This is wrong, remove this comment.

Comment (humphd) [reply server.js]: Thanks, I totally missed this!

**Comment by humphd** [comment]
Fixed the formatting I missed before and pushed a new commit.

**Comment by apetov** [comment]
@humphd Thank you for the commit!

**Pull request merged**